src/sentry/models/auditlogentry.py

```diff
@@ -93,7 +93,7 @@ def as_event(self) -> AuditLogEvent:
             organization_id=int(
                 self.organization.id
             ),  # prefer raising NoneType here over actually passing through
-            time_of_creation=self.datetime or timezone.now(),
+            date_added=self.datetime or timezone.now(),
```

Comment:
@markstory From your feedback in a previous PR lol.  This is safe, this dataclass is not being actively transmitted anywhere atm.